src/emitter.rs

```diff
@@ -59,6 +59,50 @@ impl AsRef<[u8]> for HtmlString {
     }
 }
 
+impl<const N: usize> PartialEq<&[u8; N]> for HtmlString {
```

Comment:
Using const generics shouldn't bump the MSRV since `src/reader.rs` is already using one:

https://github.com/untitaker/html5gum/blob/d7a7a631a8fda50ca375cee49333178b29305528/src/reader.rs#L304

(see also #82)